spec/models/scraper_spec.rb

```diff
@@ -183,6 +183,22 @@
   context "there is a scraper" do
     let(:scraper) { Scraper.new }
 
+    context "scraper has no data" do
+      describe "#has_data?" do
+        it{expect(scraper.has_data?).to eq false}
+      end
+    end
+
+    context "scraper has data" do
+      before :each do
+        allow(scraper).to receive(:sqlite_db_size).and_return(256000)
```

Comment:
You're right, and I missed that there's a method on scrapers `sqlite_total_rows` :hushed: . That might be more useful. In my local version it's definitely missing a scraper with data. Thanks!